frontend/packages/topology/src/components/graph-view/components/groups/GroupNode.tsx

```diff
@@ -124,23 +125,26 @@ const GroupNode: React.FC<GroupNodeProps> = ({
             badgeColor={badgeColor}
           />
           {title && (
-            <Tooltip
-              content={title}
-              position={TooltipPosition.top}
-              trigger="manual"
-              isVisible={textHover && shouldTruncate(title)}
-            >
-              <text
-                ref={textHoverRef}
-                className="odc-group-node__title"
-                x={LEFT_MARGIN + iconWidth + TEXT_MARGIN}
-                y={TOP_MARGIN + iconHeight}
-                textAnchor="start"
-                dy="-0.25em"
+            <g ref={textHoverRef}>
+              <Tooltip
+                content={title}
+                position={TooltipPosition.top}
+                trigger="manual"
+                isVisible={textHover && shouldTruncate(title)}
```

Comment:
Needs to pass the truncateOptions here as well as default truncate length is 20 chars and in the topology we truncate the name on 13 chars. So, tooltip is not visible if the name is less then 20 chars. 
```suggestion
                isVisible={textHover && shouldTruncate(title, truncateOptions)}
```

https://github.com/openshift/console/assets/2561818/a6687b12-e135-4b64-83e3-26a17c957a66

Reply:
Updated the PR. Thanks.